### PR TITLE
@broskoski: 'Send us feedback' faq bugs

### PR DESCRIPTION
**Pull request by xtina-starr**
closes: https://github.com/artsy/collector-experience/issues/71

I have this as WIP because it could use a spec or two. I wanted to get feedback before investing in writing the tests though.

**Remove the “Need more immediate assistance? Please contact us” option:**
screen shot 2016-12-29 at 4 31 19 pm

**Selects the 'Selling On Artsy' page by default when clicked launched from the 'Send us feedback':**
feedback default page

### CODE REVIEW

File: components/multi_page/view.coffee [hunk 1]
```diff
@@ -10,11 +10,11 @@ module.exports = class MultiPageView extends Backbone.View
   events:
     'click .js-page': 'change'
 
-  initialize: ({ @title, @description, @pages }) ->
+  initialize: ({ @title, @description, @pages, @defaultPage }) ->
     @collection = new Backbone.Collection _.map @pages, (id, title) ->
       new Page id: id, title: title
 
-    @state = new Backbone.Model active: @collection.first().id
+    @state = new Backbone.Model active: if @defaultPage then @defaultPage else @collection.first().id
```
Comment (xtina-starr) [inline components/multi_page/view.coffee]: Wondering if I'm being too presumptuous that the default page id passed is actually within the collection? Should I check that `@defaultPage` has a value and that it's included in the collection with the `findWhere` method or something?

Also thinking I should probably change `@defaultPage` to `@defaultPageId` to be a little more clear. 

Let me know your thoughts?

Comment (broskoski) [inline components/multi_page/view.coffee]: Yeah `defaultPageId` makes more sense, and yes agreed it should probably look for the item in the collection. However if you are passing in the id, its pretty simple:
```coffeescript
@collection.get(@defaultPageId)
````

**Comment by broskoski** [comment]
LGTM, other than the lack of specs!

File: components/multi_page/view.coffee [hunk 2]
```diff
@@ -10,11 +10,11 @@ module.exports = class MultiPageView extends Backbone.View
   events:
     'click .js-page': 'change'
 
-  initialize: ({ @title, @description, @pages, @defaultPage }) ->
+  initialize: ({ @title, @description, @pages, @defaultPageId }) ->
     @collection = new Backbone.Collection _.map @pages, (id, title) ->
       new Page id: id, title: title
 
-    @state = new Backbone.Model active: if @defaultPage then @defaultPage else @collection.first().id
+    @state = new Backbone.Model active: if @collection.get(@defaultPageId) then @defaultPageId else @collection.first().id
```
Comment (broskoski) [inline components/multi_page/view.coffee]: 👍

**Comment by broskoski** [comment]
Works for me :)

**Pull request merged**